File: packages/live-update/ios/Plugin/Classes/Options/SetNextBundleOptions.swift
import Foundation
import Capacitor

@objc public class SetNextBundleOptions: NSObject {
    private var bundleId: String?

    init(_ call: CAPPluginCall) {
        self.bundleId = call.getString("bundleId")
    }

    func getBundleId() -> String? {
        return bundleId
    }
}

File: packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/classes/options/SetNextBundleOptions.java
package io.capawesome.capacitorjs.plugins.liveupdate.classes.options;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import com.getcapacitor.PluginCall;

public class SetNextBundleOptions {

    @Nullable
    private final String bundleId;

    public SetNextBundleOptions(@NonNull PluginCall call) {
        this.bundleId = call.getString("bundleId");
    }

    @Nullable
    public String getBundleId() {
        return bundleId;
    }
}
